Build/build.msbuild

```diff
@@ -40,8 +51,12 @@
 
   <Target Name="Test" DependsOnTargets="Build">
     <Exec Command="$(OutDir)\signalrclienttests.exe --gtest_output=xml:$(OutDir)\test_results.xml" />
+    <ExecAsync Executable="$(SolutionDir)test\signalrclient-testhost\bin\$(Configuration)\signalrclient-testhost.exe" Arguments="180000" />    
```

Comment:
Make this timeout smaller (like a minute or so - currently running the tests takes less than a second)